docs/guides/python/integration_tests.md

```diff
@@ -0,0 +1,64 @@
+## :fontawesome-solid-flask-vial: Running integration tests
+
+For Python package-based projects it's pretty easy to setup the integration tests pipeline with the instruction below.
+
+!!! hint
+
+    For Notebook-based workflows, take a look at the [nutter project](https://github.com/microsoft/nutter).
+
+
+## :octicons-file-code-24:  Preparing the configuration
+
+* Add an entrypoint file to the `tests/entrypoint.py`
+
+```python title="tests/netrypoint.py"
```

Comment:
`netrypoint` -> `entrypoint`

Reply:
hi @theKashyap , 
this one has been fixed in later PRs, now it points to the correct one, please take a look at the up-to-date docs page.